src/api/controllers/termsAndConditionsController.js

```diff
@@ -0,0 +1,55 @@
+import statusCode from '../../helpers/constants/status.codes';
+import models from '../models';
+
+const { TermsAndConditions } = models;
+
+/**
+ * @class TermsAndConditionController
+ * @description a class to get or update terms and conditions of Authors haven
+ */
+export default class TermsAndConditionController {
+  /**
+   *
+   * @param {Object} req - Request
+   * @param {Object} res  - Response
+   * @param {Function} next -Next
+   * @returns {Object} The response object
+   */
+  static async getTermsAndConditions(req, res) {
+    const termsDocument = await TermsAndConditions.findOne({
+      where: {
+        id: req.params.id
+      }
+    });
+    return res.status(statusCode.OK).json({
+      termsDocument
+    });
+  }
+
+  /**
+   *
+   * @param {Object} req - Request
+   * @param {Object} res  - Response
+   * @param {Function} next -Next
+   * @returns {Object} The response object
```

Comment:
the same here! next is not one of the parameters

Reply:
done